.github/workflows/release.yml

```diff
@@ -97,14 +97,14 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        go: ['1.18.x']
+        go: ['1.17.x']
```

Comment:
Should we keep this on `1.18.x`?

Reply:
Yeah, I rebased and there was a conflict which I apparently misresolved.  Will fix.